openedx/features/learner_profile/views/learner_profile.py

```diff
@@ -42,10 +45,34 @@ def learner_profile(request, username):
         GET /account/profile
     """
     try:
-        return render_to_response(
+        context = learner_profile_context(request, username, request.user.is_staff)
+        # TODO: LEARNER-2554: 09/2017: Remove message and cookie logic when we no longer want this message
+        dismiss_message = False
+        if (context['own_profile'] and
+                BOOST_PROFILE_VISIBILITY.is_enabled() and
+                request.COOKIES.get('dismiss-profile-message', '') != 'True'):
+            PageLevelMessages.register_info_message(
+                request,
+                Text(_('Welcome to the new learner profile! When your profile visibility is set to "Full Profile", \
+                        other edX learners can now see your full name, the certificates you have earned, and when \
+                        you joined the edX community. You can also optionally add links to your social accounts from \
```

Comment:
This string should not contain "edX" but instead the product name should be used.